### DISCUSSION TITLE
See ConfigMaps in fullscreen without switching to browser fullscreen mode

### DISCUSSION THREAD

**Discussion by GioAc96** [mia-platform/community#110] [Ideas]
When I click on the "fullscreen" button on the ConfigMap viewer inside the Console it switches my browser to fullscreen. However, I find this annoying, since I would like the ConfigMap to just occupy the full viewport of the browser, without having the browser in fullscreen mode. This would allow me to switch between multiple tabs without exiting fullscreen. If this were the default behaviour, it would not have an impact on the ability to see the ConfigMap in real fullscreen, since the user could just manually switch the browser to fullscreen.

![image](https://user-images.githubusercontent.com/21307773/203527029-9372d099-108d-4949-9650-eb8bd17996d7.png)

**Comment by davidebianchi** [comment]
Hi @GioAc96! I totally agree with you! @albertotessarotto can we add it in the backlog of devx improvements?

**Reply by albertotessarotto** [reply]
Sure! I agree too, we will plan this activity!

**Reply by albertotessarotto** [reply]
This fullscreen mode improvement will be released in Console v10.4.0
